README.md

```diff
@@ -125,8 +140,66 @@ To update Spdx-Java-Library, the following is a very brief checklist:
 3. If there are any conversions that are needed when copying to or from the new model version, add conversion code to the `ModelCopyConverter` class.
 4. Update SpdxModelFactory unit test for the highest version check
 
-## Development Status
+## API Documentation
 
-Note: This library is currently unstable, and under development.
-Reviews, suggestions are welcome.
-Please enter an issue with any suggestions.
+Here are links to the API documentation for the family of SPDX Java libraries.
+
+"latest" points to the API documentation of the latest stable version of the library, while "dev" points to the API documentation generated every time there is an update in the library's GitHub repository.
+
+| Library | | |
```

Comment:
Headings in this table would be useful for readability. e.g.
> | Library | JavaDocs (Release) | JavaDocs (Dev) |

Reply:
I also wonder if this table is a good place to briefly mention what each library does?  To a newcomer this list of raw library names without any explanation might be overwhelming, and having to click into each one to find out what it does will be quite tedious.

Reply:
Agree. Added.